Libraries/Network/XMLHttpRequest.android.js

```diff
@@ -26,6 +26,12 @@ function convertHeadersMapToArray(headers: Object): Array<Header> {
 }
 
 class XMLHttpRequest extends XMLHttpRequestBase {
+  constructor() {
+    super();
+    // Native stack times out at 120s
+    // this.timeout = 120000;
+    this.timeout = 200000;
```

Comment:
My mistake here. It was mistakenly left from testing. I'll fix it if it's being considered for a merge, but I'm about to push the native implementation. Sorry for the sloppy code.